force-app/main/default/lwc/nksChatAuthenticationInfoV2/nksChatAuthenticationInfoV2.css

```
@@ -0,0 +1,14 @@
+.badge {
+    width: 100%;
+}
```

```suggestion
.badge, .button, .customBadge {
    width: 100%;
}
```
Fjern width fra de andre klassene